packages/test-utils/src/create-dom-event.js

```diff
@@ -24,6 +24,19 @@ const modifiers = {
   pagedown: 34
 }
 
+function getOptions(modifier, { bubbles, cancelable }, options) {
```

Comment:
Maybe make the third param part of the second? No need for a third separate parameter.

Reply:
Keeping parity with the calling method.

Reply:
After fixing the failing Karma tests, I refactored the method signatures to be more compact